src/camera.cpp

```diff
@@ -457,6 +457,8 @@ void Camera::update(LocalPlayer* player, f32 frametime, f32 busytime, f32 tool_r
 		fov_degrees = player->getZoomFOV();
 	} else {
 		fov_degrees = m_cache_fov;
+		//if the callback(see line 63) gets added then this won't need to be here.
+		m_cache_fov = g_settings->getFloat("fov");
```

Comment:
I guess this is temporary as we can't fetch this from settings this often.

Reply:
right, all it needs to do is be able to be reset when the FOV gets changed, but as there is no callback yet, this has to be here. As far as I have checked, it doesn't affect my performance at all.

Reply:
The performance impact is probably not very noticeable, but these small efforts add up.

Reply:
true, which is why the callbacks would need to be added